dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -513,6 +513,15 @@ export const FrontSection = ({
 		!!pageId &&
 		!!ajaxUrl;
 
+	const frontHasEditorialOrDesignBadge = !!badge;
+
+	// Only show the badge with a "Paid for by" label on the FIRST card of a paid front, aka as Labs front.
+	const isTheFirstContainerOnAPaidFront =
+		isOnPaidContentFront && index === 0 && !!frontBranding;
+
+	const isTheFirstContainerOnASponsoredFront =
+		!isOnPaidContentFront && index === 0 && !!frontBranding;
+
```

Comment:
Creating these booleans to improve readability as suggested by @arelra in https://github.com/guardian/dotcom-rendering/pull/8764#discussion_r1319732633. This actually made me realised a bug that was introduced in #8813 which made the front badge in Labs fronts to disappear


| Before | After |
|--------|-------|
| ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/f6a558a9-e103-4ec9-b7f6-b670fa655e62) | ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/b74b9de7-199c-4d56-91dd-497f59c6d53e) |